README.md

```
@@ -207,19 +208,18 @@ npm run publish-app
 
 # 🎁 Donate
 
-<a href="https://www.buymeacoffee.com/fossisthefuture">
+<a href="https://www.buymeacoffee.com/fossisthefuture" target="_blank">
   <img src="https://user-images.githubusercontent.com/25067102/154570688-9e143f2b-fee3-4b05-a9d2-a7a3013b2b51.png" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)